### PR TITLE
[Dashboard] bug fix- user has issue tracker permission,but can't find issue track in the task menu.

### PR DESCRIPTION
**Pull request by kongtiaowang**
Can't find issue track in the task menu if only have issue_tracker_developer permission.

### CODE REVIEW

**Comment by MounaSafiHarab** [comment]
@kongtiaowang 

without checking out your branch, if I have an issue assigned to me, I do see those in the dashboard/my tasks panel. However, there are two issue tracker permissions; one of them is needed (`issue_tracker_developer`; the one shown to be checked in the screenshot below). Not sure if this is the intended behavior, but it is currently working with this permission.

issuetrackerperm

**Comment by kongtiaowang** [comment]
@MounaSafiHarab Even you have both of them, you still can not find issue tracker panel on the dashboard.
screen shot 2017-09-14 at 10 01 25 am
screen shot 2017-09-14 at 10 01 49 am

**Comment by MounaSafiHarab** [comment]
@kongtiaowang 
this is because I had `data entry` permission which displayed the Incomplete Forms panel and as a result, the Issues assigned to me as well.
Works well now with your changes!

BUT I am not sure why the template file you committed has different permissions (775 instead of 664?)

File: modules/dashboard/test/TestPlan.md [hunk 1]
```diff
@@ -48,32 +48,35 @@
 14. Verify that if a user has 'Across all sites access candidate profiles' permission, the reports works and even
     more important to check without this permission.
     [Automate Test]
-15. Verify that if a user has 'User Management / Survey Participant Management' permission, the number of pending
+15. Verify that if a user has 'issue_tracker_reporter / issue_tracker_developer' permission,
+    the issue tracker panel should be found without this permission.
+    [Automate Test on Travis_CI]
```
Comment (MounaSafiHarab) [inline modules/dashboard/test/TestPlan.md]: @kongtiaowang 

this is confusing :)
Should the usre have either one or both? and did you mean to say the issue tracker panel should NOT be found?

**Pull request merged**